kolibri/plugins/user/assets/src/views/SignUpPage.vue

```diff
@@ -236,6 +236,14 @@
           !this.facilityIsInvalid
         );
       },
+      nextParam() {
+        // query is after hash
+        if (this.$route.query.next) {
+          return this.$route.query.next;
+        }
+        // query is before hash
+        return getUrlParameter('next');
```

Comment:
This produces an error, this functions needs to be imported from SignInPage/index.vue or added to this file

Reply:
thanks good catch

Reply:
pushed fix